File: 250620/폭탄 해제/bomb-removal.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String uCode = sc.next();
        char lColor = sc.next().charAt(0);
        int time = sc.nextInt();
        // Please write your code here.
        
        BoomClearInformation unit = new BoomClearInformation(uCode, lColor, time);

        System.out.printf("code : %s\ncolor : %c\nsecond : %d", unit.uCode, unit.lColor, unit.time);
    }
}

class BoomClearInformation {
    String uCode;
    char lColor;
    int time;

    BoomClearInformation(String uCode, char lColor, int time) {
        this.uCode = uCode;
        this.lColor = lColor;
        this.time = time;
    }
}

File: 250923/폭탄 해제/bomb-removal.py
unlock_code, wire_color, seconds = input().split()
seconds = int(seconds)

# Please write your code here.

class S:
    def __init__(self,unlock_code,wire_color,seconds) :
        self.unlock_code = unlock_code
        self.wire_color=wire_color
        self.seconds = seconds

s1 = S(unlock_code,wire_color,seconds)

print(f"code : {s1.unlock_code}\ncolor : {s1.wire_color}\nsecond : {s1.seconds}")